tests/e2e/tests/test_storage_efs.py

```diff
@@ -24,27 +38,55 @@
     create_efs_driver_sa,
     create_efs_volume,
     static_provisioning,
+    dynamic_provisioning,
+)
+from e2e.utils.constants import (
+    DEFAULT_USER_NAMESPACE,
+    DEFAULT_SYSTEM_NAMESPACE,
+)
+from e2e.utils.utils import (
+    unmarshal_yaml,
+    rand_name,
+    wait_for_kfp_run_succeeded_from_run_id,
 )
+from e2e.resources.pipelines.pipeline_read_from_volume import read_from_volume_pipeline
+from e2e.resources.pipelines.pipeline_write_to_volume import write_to_volume_pipeline
 
 GENERIC_KUSTOMIZE_MANIFEST_PATH = "../../docs/deployment/vanilla"
+DISABLE_PIPELINE_CACHING_PATCH_FILE = (
+    "./resources/custom-resource-templates/patch-disable-pipeline-caching.yaml"
+)
+MOUNT_PATH = "/home/jovyan/"
 
 
 @pytest.fixture(scope="class")
 def kustomize_path():
     return GENERIC_KUSTOMIZE_MANIFEST_PATH
 
 
-class TestEFS:
+class TestEFS_Static:
     @pytest.fixture(scope="class")
-    def setup(self, metadata, kustomize, static_provisioning):
+    def setup(self, metadata, kustomize, port_forward, static_provisioning):
+        # Disable caching in KFP
+        # By default KFP will cache previous pipeline runs and subsequent runs will skip cached steps
+        # This prevents artifacts from being uploaded to s3 for subsequent runs
+        patch_body = unmarshal_yaml(DISABLE_PIPELINE_CACHING_PATCH_FILE)
```

Comment:
mind moving to a common fixture/method instead of copying?

Reply:
fair point, let me do that.